website/static/js/share/utils.js

```diff
@@ -212,11 +212,7 @@ utils.loadStats = function(vm){
             if(type === 'shareDonutGraph') {
                 var count = data.charts.shareDonutGraph.columns.filter(function(val){return val[1] > 0;}).length;
                 $('.c3-chart-arcs-title').text(count + ' Provider' + (count !== 1 ? 's' : ''));
-                vm.statsData.charts.shareDonutGraph.columns = vm.statsData.charts[type].columns.filter(function(datum) {
-                    return (datum[1] > 0);
-                });
             }
-            vm.statsData.charts[type].unload = unload;
```

Comment:
You should also delete where unload is defined as well, since it is not longer used.